### PR TITLE
adding Map component

### PR DESCRIPTION
**Pull request by ShiroUsagi-san**
Adding a map component to show on a map the insalan postal address

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -44,27 +44,5 @@ const { tournaments } = storeToRefs(tournamentStore);
 </template>
 
 <style>
```
Comment (Dyskal) [inline src/views/Home.vue:46]: remove empty style tag

Comment (KwikKill) [inline src/views/Home.vue:46]: Style tag is empty so we can maybe remove it

File: index.html [hunk 2]
```diff
@@ -4,6 +4,7 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
   <meta name="description" content="Insalan">
+  <link rel="stylesheet" type="text/css" href="/src/assets/style.css" media="screen" />
```
Comment (Dyskal) [inline index.html:7]: critical css can go on global (read as non-scoped) <style> tag on App.vue

File: src/assets/style.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.title {
```
Comment (Dyskal) [inline src/assets/style.css:1]: src/style.css and src/assets/style.css might be confusing

**Review by Dyskal — CHANGES_REQUESTED** [review]

File: src/assets/style.css (hunk 3 above)
Comment (ShiroUsagi-san) [reply src/assets/style.css]: src/assets/style.css removed in [c957e31](https://github.com/InsaLan/frontend-insalan.fr/pull/40/commits/c957e3197aa63677e2c639b39ac1f1cd63997b64)

File: index.html (hunk 2 above)
Comment (ShiroUsagi-san) [reply index.html]: solved in [67b5b74](https://github.com/InsaLan/frontend-insalan.fr/pull/40/commits/67b5b7444523a779cb437504c7abb6ab7f2a3c8d)

**Review by Dyskal — APPROVED** [review]

**Review by KwikKill — APPROVED** [review]

**Pull request merged**